### PR TITLE
time slice alert by group

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### Additional notes

**Ready for review, wait to merge once we release feature**

### CODE REVIEW

File: content/en/service_management/service_level_objectives/burn_rate.md [hunk 1]
```diff
@@ -102,6 +102,10 @@ $$\text"burn rate" = {7 \text"days" * 24 \text"hours" * 10% \text"error budget c
 5. Add [Notification information][4] into the **Configure notifications and automations** section.
 6. Click the **Save and Exit** button on the SLO configuration page.
 
+### Alerting for SLOs with groups
+
+For time slice SLOs containing groups, you can set burn rate alerts based on the SLO groups or the overall SLO. If you alert based on the groups, you can confiure the [alert aggregation][8] to use simple or multi alerts. For metric-based and monitor-based SLOs, you can only set burn rate alerts based on the overall SLO.
```
Comment (maycmlee) [inline content/en/service_management/service_level_objectives/burn_rate.md:107]:
```suggestion
For Time Slice SLOs containing groups, you can set burn rate alerts based on the SLO groups or the overall SLO. If you alert based on the groups, you can confiure the [alert aggregation][8] to use simple or multi alerts. For metric-based and monitor-based SLOs, you can only set burn rate alerts based on the overall SLO.
```

File: content/en/service_management/service_level_objectives/error_budget.md [hunk 2]
```diff
@@ -35,6 +35,11 @@ over the past `target` number of days.
 
 {{< img src="service_management/service_level_objectives/slo_create_set_alert.png" alt="Create SLO and set up an error budget alert" style="width:80%;">}}
 
+### Alerting for SLOs with groups
+
+For time slice SLOs containing groups, you can set error budget alerts based on the SLO groups or the overall SLO. If you alert based on the groups, you can confiure the [alert aggregation][9] to use simple or multi alerts. For metric-based and monitor-based SLOs, you can only set error budget alerts based on the overall SLO.
```
Comment (maycmlee) [inline content/en/service_management/service_level_objectives/error_budget.md:40]:
```suggestion
For Time Slice SLOs containing groups, you can set error budget alerts based on the SLO groups or the overall SLO. If you alert based on the groups, you can confiure the [alert aggregation][9] to use simple or multi alerts. For metric-based and monitor-based SLOs, you can only set error budget alerts based on the overall SLO.
```

**Review by maycmlee — COMMENTED** [review]
Just a couple of small edits.

**Review by maycmlee — APPROVED** [review]
Thanks @roxanne-moslehi! Feel free to `/merge` if this is ready for merge.

**Comment by maycmlee** [comment]
@roxanne-moslehi is this good to merge?

**Comment by roxanne-moslehi** [comment]
We are still waiting to release the feature. I will follow up when it's ready to merge.

**Comment by roxanne-moslehi** [comment]
@maycmlee this is good to merge! fyi @zhengshizhao @peachprosecutor

**Pull request merged**